cmd/expiration-mailer/main.go

```diff
@@ -485,11 +500,21 @@ type Config struct {
 		DB cmd.DBConfig
 		cmd.SMTPConfig
 
-		From    string
+		// The "From" address for reminder messages.
+		From string
+
+		// The Subject line of reminder messages.
+		// This is a Go template with a single variable: ExpirationSubject,
+		// which contains a list of affectd hostnames, possible truncated.
 		Subject string
 
+		// Maximum number of certificates to investigate in a single batch.
 		CertLimit int
-		NagTimes  []string
+
+		// Maximum number of rows to update in a single SQL UPDATE statement.
+		UpdateChunkSize int
```

Comment:
Since we know the max parameters that MariaDB will accept in a single prepared statement we should probably check the value of this field and refuse to start the expiration-mailer if it's exceeded.